src/components/SectionTable/CourseInfoButton.js

```diff
@@ -1,26 +1,45 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { Button } from '@material-ui/core';
+import { useMediaQuery } from '@material-ui/core';
 
-class CourseInfoButton extends PureComponent {
-    render() {
-        const { text, icon, redirectLink } = this.props;
-        return (
-            <Button
-                startIcon={icon}
-                variant="contained"
-                size="small"
-                style={{ marginRight: '4px', backgroundColor: '#385EB1', color: '#fff' }}
-                onClick={(event) => {
-                    window.open(redirectLink);
-                }}
-            >
-                {text}
-            </Button>
-        );
-    }
+function CourseInfoButton({ text, icon, redirectLink }) {
+    const isMobileScreen = useMediaQuery('max-width: 750px)');
+
+    return (
+        <Button
+            startIcon={!isMobileScreen && icon}
+            variant="contained"
+            size="small"
+            style={{ marginRight: '4px', backgroundColor: '#385EB1', color: '#fff' }}
```

Comment:
While you're working on this, could you make this style a separate class? You can look at other files to see how we do styles

Reply:
I tried to make style into a class but it keeps giving me this error in the console and a blank white page after I do a search.
![2022-03-26](https://user-images.githubusercontent.com/90677583/160267985-eadc0b6d-62e4-408e-8d25-8a2492cfbdf6.png)
![2022-03-26 (1)](https://user-images.githubusercontent.com/90677583/160267989-a6f2ac13-b3ce-454b-ba25-f6f8adbbcc4f.png)

Reply:
We can try to debug more later, but for now try this:

- Add `classes` as prop. 
function CourseInfoButton ( { <code>classes</code>, text, icon, redirectLink }) {

- Delete line 17.
```diff
- const { classes } = { text, icon, redirectLink };
```